src/js/jsx/sections/libraries/assets/Graphic.jsx

```diff
@@ -51,7 +51,7 @@ define(function (require, exports, module) {
             }).bind(this).then(function (path) {
                 // Path is undefined if the graphic asset is empty (e.g. empty artboard).
                 if (path) {
-                    this.setState({ renditionPath: path });
+                    this.setState({ renditionPath: path + "?cachebuster=" + Math.random() });
```

Comment:
The URL param will invalidate browser cache to insure showing updated preview image. However, using `Math.random()` will always trigger the invalidation, so maybe we can use `element.modified` instead?